pypsa-to-imac

```diff
@@ -0,0 +1,400 @@
+                 n.links.p_nom_opt.filter(like ='gas CHP').filter(like =country)).sum()
+            var['Installed capacity|Electricity|Gases|Fossil|Natural gas'] += 0.001* (n.links.efficiency.filter(like ='OCGT').filter(like =country)*\
+                 n.links.p_nom_opt.filter(like ='OCGT').filter(like =country)).sum()
+            var['Installed capacity|Electricity|Gases|Fossil|Natural gas'] += 0.001* (n.links.efficiency.filter(like ='CCGT').filter(like =country)*\
+                 n.links.p_nom_opt.filter(like ='CCGT').filter(like =country)).sum()
+            
+            var['Installed capacity|Electricity|Gases|Fossil|Natural gas|CCS'] = 0.001*(n.links.efficiency.filter(like ='gas CHP CC').filter(like =country)*\
+                 n.links.p_nom_opt.filter(like ='gas CHP CC').filter(like =country)).sum()
+            
+            #Capacity :Biomass (CCS)
+                                                                 
+            var['Installed capacity|Electricity|Solid bio and waste|Primary solid biomass'] = 0.001*(n.links.efficiency.filter(like ='solid biomass CHP').filter(like =country)*\
+                 n.links.p_nom_opt.filter(like ='solid biomass CHP').filter(like =country)).sum() 
+            var['Installed capacity|Electricity|Solid bio and waste'] = var['Installed capacity|Electricity|Solid bio and waste|Primary solid biomass']
+            var['Installed capacity|Electricty|Biomass'] = var['Installed capacity|Electricity|Solid bio and waste']
+            
+            var['Installed capacity|Electricity|Solid bio and waste|Primary solid biomass|CCS'] = 0.001*(n.links.efficiency.filter(like ='solid biomass CHP CC').filter(like =country)*\
+                 n.links.p_nom_opt.filter(like ='solid biomass CHP CC').filter(like =country)).sum()            
+            var['Installed capacity|Electricity|Solid bio and waste|CCS'] = var['Installed capacity|Electricity|Solid bio and waste|Primary solid biomass|CCS']
+            var['Installed capacity|Electricty|Biomass|CCS'] = var['Installed capacity|Electricity|Solid bio and waste|CCS']            
+           
+            #Capacity : Hydrogen
+            
+            var['Installed capacity|Electricity|Gases|Hydrogen'] = 0.001*(n.links.efficiency.filter(like ='H2 Fuel Cell').filter(like =country)*\
+                 n.links.p_nom_opt.filter(like ='H2 Fuel Cell').filter(like =country)).sum()
+
+            #Capacity : Oil 
+
+            var['Installed capacity|Electricity|Liquids|Fossil'] = 0.001*(n.links.efficiency.filter(like ='oil-').filter(like =country)*\
+                 n.links.p_nom_opt.filter(like ='oil-').filter(like =country)).sum()
+
+            #Capacity : Power to gas 
+            """
+            var['Installed capacity|P2G|Electricity'] = 0.001* (n.links.efficiency.filter(like ='Sabatier').filter(like =country)*\
```

Comment:
Acc. to IAM team, it means "the power capacity used to produce H2, synthetic methane, synthetic oil or the three of them". So it can include Sabatier , Helmeth (power to methane),  Electrolysis (water to hydrogen), and Fischer-Tropsch.